frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -398,6 +398,15 @@
                     vm.submissionResult = response;
                     // navigate to challenge page
                     // $state.go('web.challenge-page.overview');
+                    if (vm.submissionResult.count === 0) {
+                        vm.showPagination = false;
+                        vm.paginationMsg = "No team exist for now, Start by creating a new team!";
```

Comment:
@anweshknayak I think this message is for team creation and you have copied it from team controller. Please change it to "No results found"